index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    <title>GMP QUIZ APP</title>
+</head>
+<body>
+    <div class="app">
+        <h1>GMP QUIZ</h1>
+        <div class="quiz">
+            <h2 id="question">What is GMP</h2>
+           <div id="answerButtons">
+            <button class=" btn">Answer 1</button>
+            <button class=" btn">Answer 2</button>
+            <button class=" btn">Answer 3</button>
+            <button class=" btn">Answer 4</button>
+           </div>
+           <button id="btn-next">Next</button>
+           <button id="shuffleButton">shuffle Button</button>
```

Comment:
Usually with HTML then `kebab-case` is definitely the most common practice. Having said that I do know a developer or two who prefer Javascript's `camelCase` but the most important thing is to be consistent.

Reply:
Thank you so much!!!